Installing with CUDA fails on Bazzite

### Formula Name

cuda@13.1

### Homebrew Version

Homebrew 5.1.7

### Operating System

bazzite-deck-nvidia:stable 43.20260420

### Describe the Issue

Installing sunshine-beta with CUDA fails during the cmake process due to missing X. Bazzite does not contain X, just Wayland.

### Relevant Terminal Output

```shell
brew reinstall sunshine-beta --with-cuda
==> Fetching downloads for: sunshine-beta
✔︎ Bottle Manifest cmake (4.3.2)                                                                                                                                                                                                                                                                                                                                                                                                                                             Downloaded   11.7KB/ 11.7KB
✔︎ Bottle Manifest pkgconf (2.5.1)                                                                                                                                                                                                                                                                                                                                                                                                                                           Downloaded   12.2KB/ 12.2KB
✔︎ Bottle pkgconf (2.5.1)                                                                                                                                                                                                                                                                                                                                                                                                                                                    Downloaded  149.1KB/149.1KB
✔︎ Bottle Manifest isl (0.27)                                                                                                                                                                                                                                                                                                                                                                                                                                                Downloaded   14.5KB/ 14.5KB
✔︎ Bottle isl (0.27)                                                                                                                                                                                                                                                                                                                                                                                                                                                         Downloaded    2.3MB/  2.3MB
✔︎ Bottle Manifest mpfr (4.2.2)                                                                                                                                                                                                                                                                                                                                                                                                                                              Downloaded   12.7KB/ 12.7KB
✔︎ Bottle mpfr (4.2.2)                                                                                                                                                                                                                                                                                                                                                                                                                                                       Downloaded    1.3MB/  1.3MB
✔︎ Bottle Manifest libmpc (1.4.1)                                                                                                                                                                                                                                                                                                                                                                                                                                            Downloaded   12.0KB/ 12.0KB
✔︎ Bottle libmpc (1.4.1)                                                                                                                                                                                                                                                                                                                                                                                                                                                     Downloaded  194.2KB/194.2KB
✔︎ Bottle Manifest binutils (2.46.0)                                                                                                                                                                                                                                                                                                                                                                                                                                         Downloaded   41.0KB/ 41.0KB
✔︎ Bottle Manifest gcc@14 (14.3.0)                                                                                                                                                                                                                                                                                                                                                                                                                                           Downloaded   23.2KB/ 23.2KB
✔︎ Bottle Manifest cuda@13.1 (13.1.1)                                                                                                                                                                                                                                                                                                                                                                                                                                        Downloaded    4.2KB/  4.2KB
✔︎ Resource sunshine-beta--markupsafe                                                                                                                                                                                                                                                                                                                                                                                                                                        Verified     80.3KB/ 80.3KB
✔︎ Resource sunshine-beta--jinja2                                                                                                                                                                                                                                                                                                                                                                                                                                            Verified    245.1KB/245.1KB
✔︎ Resource sunshine-beta--setuptools                                                                                                                                                                                                                                                                                                                                                                                                                                        Verified      1.2MB/  1.2MB
✔︎ Bottle cmake (4.3.2)                                                                                                                                                                                                                                                                                                                                                                                                                                                      Downloaded   25.0MB/ 25.0MB
✔︎ Bottle binutils (2.46.0)                                                                                                                                                                                                                                                                                                                                                                                                                                                  Downloaded  100.9MB/100.9MB
✔︎ Bottle gcc@14 (14.3.0)                                                                                                                                                                                                                                                                                                                                                                                                                                                    Downloaded  147.5MB/147.5MB
✔︎ Bottle cuda@13.1 (13.1.1)                                                                                                                                                                                                                                                                                                                                                                                                                                                 Downloaded    3.9GB/  3.9GB
✔︎ Formula sunshine-beta (2026.423.21833)
==> Reinstalling lizardbyte/homebrew/sunshine-beta --with-cuda
==> Installing dependencies for lizardbyte/homebrew/sunshine-beta: cmake, pkgconf, isl, mpfr, libmpc, binutils, gcc@14 and lizardbyte/homebrew/cuda@13.1
==> Installing lizardbyte/homebrew/sunshine-beta dependency: cmake
==> Pouring cmake--4.3.2.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/cmake/4.3.2: 4,044 files, 70.7MB
==> Installing lizardbyte/homebrew/sunshine-beta dependency: pkgconf
==> Pouring pkgconf--2.5.1.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/pkgconf/2.5.1: 29 files, 509.2KB
==> Installing lizardbyte/homebrew/sunshine-beta dependency: isl
==> Pouring isl--0.27.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/isl/0.27: 75 files, 10.3MB
==> Installing lizardbyte/homebrew/sunshine-beta dependency: mpfr
==> Pouring mpfr--4.2.2.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/mpfr/4.2.2: 32 files, 4.1MB
==> Installing lizardbyte/homebrew/sunshine-beta dependency: libmpc
==> Pouring libmpc--1.4.1.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/libmpc/1.4.1: 15 files, 678.9KB
==> Installing lizardbyte/homebrew/sunshine-beta dependency: binutils
==> Pouring binutils--2.46.0.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/binutils/2.46.0: 6,289 files, 568.2MB
==> Installing lizardbyte/homebrew/sunshine-beta dependency: gcc@14
==> Pouring gcc@14--14.3.0.x86_64_linux.bottle.3.tar.gz
==> Creating the GCC specs file: /var/home/linuxbrew/.linuxbrew/Cellar/gcc@14/14.3.0/bin/../lib/gcc/14/gcc/x86_64-pc-linux-gnu/14/specs
🍺  /home/linuxbrew/.linuxbrew/Cellar/gcc@14/14.3.0: 2,011 files, 423.8MB
==> Installing lizardbyte/homebrew/sunshine-beta dependency: lizardbyte/homebrew/cuda@13.1
==> Pouring cuda@13.1--13.1.1.x86_64_linux.bottle.tar.gz
🍺  /home/linuxbrew/.linuxbrew/Cellar/cuda@13.1/13.1.1: 7,592 files, 7.5GB
==> Installing lizardbyte/homebrew/sunshine-beta --with-cuda
==> python3 -m venv --system-site-packages --without-pip /var/tmp/sunshine-beta-20260426-17592-cyp2n/venv
==> python3 -m pip --python=/var/tmp/sunshine-beta-20260426-17592-cyp2n/venv/bin/python install /var/tmp/sunshine-beta--markupsafe-20260426-17592-myx0mq/markupsafe-3.0.3
==> python3 -m pip --python=/var/tmp/sunshine-beta-20260426-17592-cyp2n/venv/bin/python install /var/tmp/sunshine-beta--jinja2-20260426-17592-hlxc4b/jinja2-3.1.6
==> python3 -m pip --python=/var/tmp/sunshine-beta-20260426-17592-cyp2n/venv/bin/python install /var/tmp/sunshine-beta--setuptools-20260426-17592-kv7jmf/setuptools-80.10.2
==> Building tests: disabled
==> Building docs: disabled
==> Disabled statically linking Boost libraries
==> CUDA disabled
==> cmake -S . -B build -G Unix Makefiles -DBUILD_WERROR=ON -DCMAKE_CXX_STANDARD=23 -DGLAD_SKIP_PIP_INSTALL=ON -DHOMEBREW_ALLOW_FETCHCONTENT=ON -DOPENSSL_ROOT_DIR=/home/linuxbrew/.linuxbrew/opt/openssl@3 -DSUNSHINE_ASSETS_DIR=sunshine/assets -DSUNSHINE_BUILD_HOMEBREW=ON -DSUNSHINE_PUBLISHER_NAME='LizardByte' -DSUNSHINE_PUBLISHER_WEBSITE='https://app.lizardbyte.dev' -DSUNSHINE_PUBLISHER_ISSUE_URL='https://app.lizardbyte.dev/support' -DPython_EXECUTABLE=/var/tmp/sunshine-beta-20260426
Last 15 lines from /home/bazzite/.cache/Homebrew/Logs/sunshine-beta/05.cmake.log:
wayland-scanner client-header /var/tmp/sunshine-beta-20260426-17592-cyp2n/third-party/wayland-protocols/unstable/linux-dmabuf/linux-dmabuf-unstable-v1.xml /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/generated-src/linux-dmabuf-unstable-v1.h
wayland-scanner private-code /var/tmp/sunshine-beta-20260426-17592-cyp2n/third-party/wlr-protocols/unstable/wlr-screencopy-unstable-v1.xml /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/generated-src/wlr-screencopy-unstable-v1.c
wayland-scanner client-header /var/tmp/sunshine-beta-20260426-17592-cyp2n/third-party/wlr-protocols/unstable/wlr-screencopy-unstable-v1.xml /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/generated-src/wlr-screencopy-unstable-v1.h
CMake Error at /home/linuxbrew/.linuxbrew/opt/cmake/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:290 (message):
  Could NOT find X11 (missing: X11_X11_INCLUDE_PATH)
Call Stack (most recent call first):
  /home/linuxbrew/.linuxbrew/opt/cmake/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:654 (_FPHSA_FAILURE_MESSAGE)
  /home/linuxbrew/.linuxbrew/opt/cmake/share/cmake/Modules/FindX11.cmake:697 (find_package_handle_standard_args)
  cmake/macros/common.cmake:27 (_find_package)
  cmake/compile_definitions/linux.cmake:225 (find_package)
  cmake/compile_definitions/common.cmake:52 (include)
  CMakeLists.txt:64 (include)


-- Configuring incomplete, errors occurred!

If reporting this issue please do so at (not Homebrew/* repositories):
  https://github.com/lizardbyte/homebrew-homebrew/issues
```

### Full Installation Log

```shell
2026-04-26 17:34:59 +0000

cmake
-S
.
-B
build
-G
Unix Makefiles
-DCMAKE_INSTALL_PREFIX=/home/linuxbrew/.linuxbrew/Cellar/sunshine-beta/2026.423.21833
-DCMAKE_INSTALL_LIBDIR=lib
-DCMAKE_BUILD_TYPE=Release
-DCMAKE_FIND_FRAMEWORK=LAST
-DCMAKE_VERBOSE_MAKEFILE=ON
-DCMAKE_PROJECT_TOP_LEVEL_INCLUDES=/var/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/cmake/trap_fetchcontent_provider.cmake
-Wno-dev
-DBUILD_TESTING=OFF
-DBUILD_WERROR=ON
-DCMAKE_CXX_STANDARD=23
-DCMAKE_INSTALL_PREFIX=/home/linuxbrew/.linuxbrew/Cellar/sunshine-beta/2026.423.21833
-DGLAD_SKIP_PIP_INSTALL=ON
-DHOMEBREW_ALLOW_FETCHCONTENT=ON
-DOPENSSL_ROOT_DIR=/home/linuxbrew/.linuxbrew/opt/openssl@3
-DSUNSHINE_ASSETS_DIR=sunshine/assets
-DSUNSHINE_BUILD_HOMEBREW=ON
-DSUNSHINE_PUBLISHER_NAME='LizardByte'
-DSUNSHINE_PUBLISHER_WEBSITE='https://app.lizardbyte.dev'
-DSUNSHINE_PUBLISHER_ISSUE_URL='https://app.lizardbyte.dev/support'
-DPython_EXECUTABLE=/var/tmp/sunshine-beta-20260426-17592-cyp2n/venv/bin/python3
-DBUILD_TESTS=OFF
-DBUILD_DOCS=OFF
-DBOOST_USE_STATIC=OFF
-DSUNSHINE_ENABLE_CUDA=OFF

-- The C compiler identification is GNU 14.3.0
-- The CXX compiler identification is GNU 14.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/linuxbrew/.linuxbrew/opt/gcc@14/bin/gcc-14 - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/linuxbrew/.linuxbrew/opt/gcc@14/bin/g++-14 - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
Got from CI '' branch and version '2026.423.21833'
Extracting year and month/day from PROJECT_VERSION: 2026.423.21833
PROJECT_FQDN: dev.lizardbyte.app.Sunshine
PROJECT_NAME: Sunshine
PROJECT_VERSION: 2026.423.21833
PROJECT_VERSION_MAJOR: 2026
PROJECT_VERSION_MINOR: 423
PROJECT_VERSION_PATCH: 21833
CMAKE_PROJECT_VERSION: 2026.423.21833
CMAKE_PROJECT_VERSION_MAJOR: 2026
CMAKE_PROJECT_VERSION_MINOR: 423
CMAKE_PROJECT_VERSION_PATCH: 21833
RC_VERSION_BUILD: 218
RC_VERSION_REVISION: 33
PROJECT_YEAR: 2026
PROJECT_MONTH: 04
PROJECT_DAY: 23
-- Found OpenSSL: /home/linuxbrew/.linuxbrew/opt/openssl@3/lib/libcrypto.so (found version "3.6.2")
CMake Warning at cmake/macros/common.cmake:27 (_find_package):
  Could not find a configuration file for package "Boost" that exactly
  matches requested version "1.89.0".

  The following configuration files were considered but not accepted:

    /home/linuxbrew/.linuxbrew/lib/cmake/Boost-1.90.0/BoostConfig.cmake, version: 1.90.0
      The version found is not compatible with the version requested.

Call Stack (most recent call first):
  cmake/dependencies/Boost_Sunshine.cmake:33 (find_package)
  cmake/dependencies/common.cmake:9 (include)
  CMakeLists.txt:61 (include)


-- Boost v1.89.0 package not found in the system. Falling back to FetchContent.
-- Boost: Release build, static libraries, MPI OFF, Python OFF, testing OFF
-- Boost: libraries included: filesystem;locale;log;program_options;system;asio;crc;format;process;property_tree
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Performing Test BOOST_ATOMIC_TARGET_X86
-- Performing Test BOOST_ATOMIC_TARGET_X86 - Success
-- Performing Test BOOST_ATOMIC_COMPILER_HAS_SSE2
-- Performing Test BOOST_ATOMIC_COMPILER_HAS_SSE2 - Success
-- Performing Test BOOST_ATOMIC_COMPILER_HAS_SSE41
-- Performing Test BOOST_ATOMIC_COMPILER_HAS_SSE41 - Success
-- Performing Test BOOST_ATOMIC_HAS_PTHREAD_COND_CLOCKWAIT
-- Performing Test BOOST_ATOMIC_HAS_PTHREAD_COND_CLOCKWAIT - Failed
-- Performing Test BOOST_CHARCONV_QUADMATH_FOUND
-- Performing Test BOOST_CHARCONV_QUADMATH_FOUND - Success
-- Boost.Charconv: quadmath support ON
-- Boost.Context: architecture x86_64, binary format elf, ABI sysv, assembler gas, suffix .S, implementation fcontext
-- The ASM compiler identification is GNU
-- Found assembler: /home/linuxbrew/.linuxbrew/opt/gcc@14/bin/gcc-14
-- Performing Test BOOST_FILESYSTEM_HAS_INIT_PRIORITY
-- Performing Test BOOST_FILESYSTEM_HAS_INIT_PRIORITY - Success
-- Performing Test BOOST_FILESYSTEM_HAS_CXX20_ATOMIC_REF
-- Performing Test BOOST_FILESYSTEM_HAS_CXX20_ATOMIC_REF - Failed
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_BLKSIZE
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_BLKSIZE - Success
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_MTIM
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_MTIM - Success
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_MTIMENSEC
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_MTIMENSEC - Failed
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_MTIMESPEC
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_MTIMESPEC - Failed
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_BIRTHTIMENSEC
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_BIRTHTIMENSEC - Failed
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_BIRTHTIMESPEC
-- Performing Test BOOST_FILESYSTEM_HAS_STAT_ST_BIRTHTIMESPEC - Failed
-- Performing Test BOOST_FILESYSTEM_HAS_STATX
-- Performing Test BOOST_FILESYSTEM_HAS_STATX - Success
-- Performing Test BOOST_FILESYSTEM_HAS_FDOPENDIR_NOFOLLOW
-- Performing Test BOOST_FILESYSTEM_HAS_FDOPENDIR_NOFOLLOW - Success
-- Performing Test BOOST_FILESYSTEM_HAS_DIRENT_D_TYPE
-- Performing Test BOOST_FILESYSTEM_HAS_DIRENT_D_TYPE - Success
-- Performing Test BOOST_FILESYSTEM_HAS_POSIX_AT_APIS
-- Performing Test BOOST_FILESYSTEM_HAS_POSIX_AT_APIS - Success
-- Performing Test BOOST_FILESYSTEM_HAS_FALLOCATE
-- Performing Test BOOST_FILESYSTEM_HAS_FALLOCATE - Success
-- Performing Test Iconv_IS_BUILT_IN
-- Performing Test Iconv_IS_BUILT_IN - Success
-- Found Iconv: built in to C library
-- Found ICU: /home/linuxbrew/.linuxbrew/include (found suitable version "78.3", minimum required is "4.8.1") found components: data i18n uc
-- Boost.Locale: iconv ON, ICU ON, POSIX ON, std ON, winapi OFF
-- Performing Test BOOST_LOG_TARGET_X86
-- Performing Test BOOST_LOG_TARGET_X86 - Success
-- Performing Test BOOST_LOG_HAS_LOCK_FREE_ATOMIC_INT32
-- Performing Test BOOST_LOG_HAS_LOCK_FREE_ATOMIC_INT32 - Success
-- Performing Test BOOST_LOG_COMPILER_HAS_SSSE3
-- Performing Test BOOST_LOG_COMPILER_HAS_SSSE3 - Success
-- Performing Test BOOST_LOG_COMPILER_HAS_AVX2
-- Performing Test BOOST_LOG_COMPILER_HAS_AVX2 - Success
-- Performing Test BOOST_LOG_HAS_NATIVE_SYSLOG
-- Performing Test BOOST_LOG_HAS_NATIVE_SYSLOG - Success
-- Found ICU: /home/linuxbrew/.linuxbrew/include (found version "78.3") found components: data i18n uc
-- Boost.Thread: threading API is pthread
-- Boost include dirs: $<BUILD_INTERFACE:/var/tmp/sunshine-beta-20260426-17592-cyp2n/build/_deps/boost-src/libs/headers/include>
-- Boost libraries: Boost::filesystem;Boost::locale;Boost::log;Boost::program_options;Boost::system;Boost::asio;Boost::crc;Boost::format;Boost::process;Boost::property_tree
-- Looking for fcntl
-- Looking for fcntl - found
-- Looking for ioctl
-- Looking for ioctl - found
-- Looking for poll
-- Looking for poll - found
-- Looking for getaddrinfo
-- Looking for getaddrinfo - found
-- Looking for getnameinfo
-- Looking for getnameinfo - found
-- Looking for gethostbyname_r
-- Looking for gethostbyname_r - found
-- Looking for gethostbyaddr_r
-- Looking for gethostbyaddr_r - found
-- Looking for inet_pton
-- Looking for inet_pton - found
-- Looking for inet_ntop
-- Looking for inet_ntop - found
-- Performing Test HAS_MSGHDR_FLAGS
-- Performing Test HAS_MSGHDR_FLAGS - Success
-- Check size of socklen_t
-- Check size of socklen_t - done
-- Check size of QOS_FLOWID
-- Check size of QOS_FLOWID - failed
-- Check size of PQOS_FLOWID
-- Check size of PQOS_FLOWID - failed
-- nlohmann_json v3.11.x package not found in the system. Falling back to FetchContent.
-- Using the multi-header code from /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/_deps/json-src/include/
CMake Warning at third-party/libdisplaydevice/src/CMakeLists.txt:14 (message):
  Linux is not supported yet.


-- nlohmann_json v3.11.x package not found in the system. Falling back to FetchContent.
-- Found PkgConfig: /home/linuxbrew/.linuxbrew/bin/pkg-config (found version "2.5.1")
-- Checking for module 'libcurl'
--   Found libcurl, version 8.19.0
-- Checking for module 'miniupnpc'
--   Found miniupnpc, version 2.3.3
-- Using FFmpeg from build-deps tag: v2026.323.141148
-- Downloading FFmpeg binaries from https://github.com/LizardByte/build-deps/releases/download/v2026.323.141148/Linux-x86_64-ffmpeg.tar.gz
-- [download 0% complete]
-- [download 1% complete]
-- [download 2% complete]
-- [download 3% complete]
-- [download 4% complete]
-- [download 5% complete]
-- [download 6% complete]
-- [download 7% complete]
-- [download 8% complete]
-- [download 9% complete]
-- [download 10% complete]
-- [download 11% complete]
-- [download 12% complete]
-- [download 13% complete]
-- [download 14% complete]
-- [download 15% complete]
-- [download 16% complete]
-- [download 17% complete]
-- [download 18% complete]
-- [download 19% complete]
-- [download 20% complete]
-- [download 21% complete]
-- [download 22% complete]
-- [download 23% complete]
-- [download 24% complete]
-- [download 25% complete]
-- [download 26% complete]
-- [download 27% complete]
-- [download 28% complete]
-- [download 29% complete]
-- [download 30% complete]
-- [download 31% complete]
-- [download 32% complete]
-- [download 33% complete]
-- [download 34% complete]
-- [download 35% complete]
-- [download 36% complete]
-- [download 37% complete]
-- [download 38% complete]
-- [download 39% complete]
-- [download 40% complete]
-- [download 41% complete]
-- [download 42% complete]
-- [download 43% complete]
-- [download 44% complete]
-- [download 45% complete]
-- [download 46% complete]
-- [download 47% complete]
-- [download 48% complete]
-- [download 49% complete]
-- [download 50% complete]
-- [download 51% complete]
-- [download 52% complete]
-- [download 53% complete]
-- [download 54% complete]
-- [download 55% complete]
-- [download 56% complete]
-- [download 57% complete]
-- [download 58% complete]
-- [download 59% complete]
-- [download 60% complete]
-- [download 61% complete]
-- [download 62% complete]
-- [download 63% complete]
-- [download 64% complete]
-- [download 65% complete]
-- [download 66% complete]
-- [download 67% complete]
-- [download 68% complete]
-- [download 69% complete]
-- [download 70% complete]
-- [download 71% complete]
-- [download 72% complete]
-- [download 73% complete]
-- [download 74% complete]
-- [download 75% complete]
-- [download 76% complete]
-- [download 77% complete]
-- [download 78% complete]
-- [download 79% complete]
-- [download 80% complete]
-- [download 81% complete]
-- [download 82% complete]
-- [download 83% complete]
-- [download 84% complete]
-- [download 85% complete]
-- [download 86% complete]
-- [download 87% complete]
-- [download 88% complete]
-- [download 89% complete]
-- [download 90% complete]
-- [download 91% complete]
-- [download 92% complete]
-- [download 93% complete]
-- [download 94% complete]
-- [download 95% complete]
-- [download 96% complete]
-- [download 97% complete]
-- [download 98% complete]
-- [download 99% complete]
-- [download 100% complete]
-- Extracting FFmpeg binaries to /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/_deps
-- FFmpeg binaries successfully downloaded and extracted
-- Found Opus: /home/linuxbrew/.linuxbrew/lib/libopus.so
-- glad: using provided Python interpreter: /var/tmp/sunshine-beta-20260426-17592-cyp2n/venv/bin/python3
-- Glad Library 'glad_egl'
-- Found Python: /var/tmp/sunshine-beta-20260426-17592-cyp2n/venv/bin/python3 (found version "3.14.4") found components: Interpreter
-- Glad Library 'glad_gl'
-- Checking for module 'libdrm'
--   Found libdrm, version 2.4.131
-- Found LIBDRM: /home/linuxbrew/.linuxbrew/lib/libdrm.so
-- Checking for module 'libcap'
--   Found libcap, version 2.78
-- Found LIBCAP: /home/linuxbrew/.linuxbrew/lib/libcap.so
-- Checking for module 'libevdev'
--   Package 'libevdev' not found
-- libevdev source dir: /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/libevdev-1.13.2/src/libevdev
-- libevdev build dir: /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/libevdev-1.13.2/src/libevdev
-- Found libevdev library: /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/libevdev-1.13.2/src/libevdev/libevdev/.libs/libevdev.a
-- Found libevdev include directory: /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/libevdev-1.13.2/src/libevdev
-- Checking for module 'libva'
--   Found libva, version 1.23.0
-- Checking for module 'libva-drm'
--   Found libva-drm, version 1.23.0
-- Found Libva: /home/linuxbrew/.linuxbrew/include
-- Found Wayland: /home/linuxbrew/.linuxbrew/lib/libwayland-client.so;/home/linuxbrew/.linuxbrew/lib/libwayland-server.so;/home/linuxbrew/.linuxbrew/lib/libwayland-egl.so;/home/linuxbrew/.linuxbrew/lib/libwayland-cursor.so
wayland-scanner private-code /var/tmp/sunshine-beta-20260426-17592-cyp2n/third-party/wayland-protocols/unstable/xdg-output/xdg-output-unstable-v1.xml /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/generated-src/xdg-output-unstable-v1.c
wayland-scanner client-header /var/tmp/sunshine-beta-20260426-17592-cyp2n/third-party/wayland-protocols/unstable/xdg-output/xdg-output-unstable-v1.xml /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/generated-src/xdg-output-unstable-v1.h
wayland-scanner private-code /var/tmp/sunshine-beta-20260426-17592-cyp2n/third-party/wayland-protocols/unstable/linux-dmabuf/linux-dmabuf-unstable-v1.xml /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/generated-src/linux-dmabuf-unstable-v1.c
wayland-scanner client-header /var/tmp/sunshine-beta-20260426-17592-cyp2n/third-party/wayland-protocols/unstable/linux-dmabuf/linux-dmabuf-unstable-v1.xml /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/generated-src/linux-dmabuf-unstable-v1.h
wayland-scanner private-code /var/tmp/sunshine-beta-20260426-17592-cyp2n/third-party/wlr-protocols/unstable/wlr-screencopy-unstable-v1.xml /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/generated-src/wlr-screencopy-unstable-v1.c
wayland-scanner client-header /var/tmp/sunshine-beta-20260426-17592-cyp2n/third-party/wlr-protocols/unstable/wlr-screencopy-unstable-v1.xml /var/tmp/sunshine-beta-20260426-17592-cyp2n/build/generated-src/wlr-screencopy-unstable-v1.h
CMake Error at /home/linuxbrew/.linuxbrew/opt/cmake/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:290 (message):
  Could NOT find X11 (missing: X11_X11_INCLUDE_PATH)
Call Stack (most recent call first):
  /home/linuxbrew/.linuxbrew/opt/cmake/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:654 (_FPHSA_FAILURE_MESSAGE)
  /home/linuxbrew/.linuxbrew/opt/cmake/share/cmake/Modules/FindX11.cmake:697 (find_package_handle_standard_args)
  cmake/macros/common.cmake:27 (_find_package)
  cmake/compile_definitions/linux.cmake:225 (find_package)
  cmake/compile_definitions/common.cmake:52 (include)
  CMakeLists.txt:64 (include)


-- Configuring incomplete, errors occurred!

HOMEBREW_VERSION: 5.1.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 3aae056b8d072624255bc8fd27febb7f327b2265
Last commit: 9 days ago
Branch: stable
Core tap JSON: 26 Apr 17:15 UTC
HOMEBREW_PREFIX: /home/linuxbrew/.linuxbrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: :0
HOMEBREW_DOWNLOAD_CONCURRENCY: 32
HOMEBREW_EDITOR: /usr/bin/vim
HOMEBREW_MAKE_JOBS: 16
HOMEBREW_NO_AUTO_UPDATE: set
HOMEBREW_NO_BOOTSNAP: set
SUDO_ASKPASS: /usr/bin/ksshaskpass
Homebrew Ruby: 4.0.2 => /var/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/vendor/portable-ruby/4.0.2_1/bin/ruby
CPU: 16-core 64-bit zen5
Clang: 22.1.4
Git: 2.53.0 => /bin/git
Curl: 8.19.0 => /home/linuxbrew/.linuxbrew/opt/curl/bin/curl
Kernel: Linux 6.17.7-ba29.fc43.x86_64 x86_64 GNU/Linux
OS: Bazzite (Kinoite)
Host glibc: 2.42
Host libstdc++: 6.0.34
/usr/bin/gcc: 15.2.1
/usr/bin/ruby: N/A
glibc: N/A
gcc@12: N/A
gcc: N/A
xorg: N/A

CC: /home/linuxbrew/.linuxbrew/opt/gcc@14/bin/gcc-14
CXX: /home/linuxbrew/.linuxbrew/opt/gcc@14/bin/g++-14
OBJC: gcc-14
OBJCXX: g++-14
HOMEBREW_CC: gcc-14
HOMEBREW_CXX: g++-14
MAKEFLAGS: -j16
CMAKE_PREFIX_PATH: /home/linuxbrew/.linuxbrew/opt/systemd:/home/linuxbrew/.linuxbrew
PKG_CONFIG_PATH: /home/linuxbrew/.linuxbrew/opt/pkgconf/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/icu4c@78/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/xz/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/lz4/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/zlib-ng-compat/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/zstd/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/bzip2/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/brotli/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libnghttp2/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libnghttp3/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/openssl@3/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libngtcp2/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libssh2/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libidn2/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/keyutils/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/ncurses/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libedit/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/krb5/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/readline/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxcrypt/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/sqlite/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/util-linux/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/cyrus-sasl/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/openldap/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/curl/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/miniupnpc/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/opus/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/python@3.14/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/expat/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/dbus/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/pcre2/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libffi/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/mpdecimal/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/glib/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxau/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxdmcp/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxcb/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libx11/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxext/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxfixes/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxi/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxtst/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/at-spi2-core/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libcap/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libdaemon/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/systemd/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/avahi/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libpng/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/freetype/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/fontconfig/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxrender/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/lzo/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/pixman/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/cairo/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/jpeg-turbo/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libtiff/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxml2/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/gdk-pixbuf/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/fribidi/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/graphite2/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/harfbuzz/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxrandr/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxxf86vm/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/z3/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/spirv-llvm-translator/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/spirv-tools/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/xcb-util-keysyms/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libpciaccess/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libdrm/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxshmfence/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/wayland/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/elfutils/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/mesa/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/mesa-glu/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxinerama/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/freeglut/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libepoxy/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libdatrie/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libthai/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/pango/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxdamage/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libfontenc/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxfont2/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libice/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libsm/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxt/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxmu/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/xcb-util/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/xcb-util-image/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/xcb-util-renderutil/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/xcb-util-wm/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxkbfile/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/xkbcomp/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libtirpc/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxcvt/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/gmp/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/nettle/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/xorg-server/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxkbcommon/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/gtk+3/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/ayatana-ido/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libayatana-indicator/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/json-glib/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libdbusmenu/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libayatana-appindicator/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libnotify/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libva/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libxcursor/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/numactl/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/alsa-lib/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/fftw/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libogg/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/flac/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libvorbis/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/mpg123/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libsndfile/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libsoxr/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/orc/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/speexdsp/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/pulseaudio/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/pipewire/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/shaderc/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/vulkan-loader/lib/pkgconfig:/home/linuxbrew/.linuxbrew/opt/xorgproto/share/pkgconfig:/home/linuxbrew/.linuxbrew/opt/systemd/share/pkgconfig:/home/linuxbrew/.linuxbrew/opt/shared-mime-info/share/pkgconfig:/home/linuxbrew/.linuxbrew/opt/gsettings-desktop-schemas/share/pkgconfig:/home/linuxbrew/.linuxbrew/opt/hicolor-icon-theme/share/pkgconfig:/home/linuxbrew/.linuxbrew/opt/libclc/share/pkgconfig:/home/linuxbrew/.linuxbrew/opt/iso-codes/share/pkgconfig:/home/linuxbrew/.linuxbrew/opt/xkeyboard-config/share/pkgconfig:/home/linuxbrew/.linuxbrew/opt/wayland-protocols/share/pkgconfig
PKG_CONFIG_LIBDIR: 
HOMEBREW_MAKE_JOBS: 16
HOMEBREW_GIT: git
ACLOCAL_PATH: /home/linuxbrew/.linuxbrew/share/aclocal
PATH: /var/home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/shims/linux/super:/home/linuxbrew/.linuxbrew/opt/cmake/bin:/home/linuxbrew/.linuxbrew/opt/node/bin:/home/linuxbrew/.linuxbrew/opt/pkgconf/bin:/home/linuxbrew/.linuxbrew/opt/icu4c@78/bin:/home/linuxbrew/.linuxbrew/opt/xz/bin:/home/linuxbrew/.linuxbrew/opt/lz4/bin:/home/linuxbrew/.linuxbrew/opt/zstd/bin:/home/linuxbrew/.linuxbrew/opt/bzip2/bin:/home/linuxbrew/.linuxbrew/opt/brotli/bin:/home/linuxbrew/.linuxbrew/opt/openssl@3/bin:/home/linuxbrew/.linuxbrew/opt/libidn2/bin:/home/linuxbrew/.linuxbrew/opt/keyutils/bin:/home/linuxbrew/.linuxbrew/opt/ncurses/bin:/home/linuxbrew/.linuxbrew/opt/krb5/bin:/home/linuxbrew/.linuxbrew/opt/sqlite/bin:/home/linuxbrew/.linuxbrew/opt/util-linux/bin:/home/linuxbrew/.linuxbrew/opt/openldap/bin:/home/linuxbrew/.linuxbrew/opt/curl/bin:/home/linuxbrew/.linuxbrew/opt/miniupnpc/bin:/home/linuxbrew/.linuxbrew/opt/gcc@14/bin:/home/linuxbrew/.linuxbrew/opt/cuda@13.1/bin:/home/linuxbrew/.linuxbrew/opt/python@3.14/bin:/home/linuxbrew/.linuxbrew/opt/expat/bin:/home/linuxbrew/.linuxbrew/opt/dbus/bin:/home/linuxbrew/.linuxbrew/opt/pcre2/bin:/home/linuxbrew/.linuxbrew/opt/unzip/bin:/home/linuxbrew/.linuxbrew/opt/glib/bin:/home/linuxbrew/.linuxbrew/opt/gdbm/bin:/home/linuxbrew/.linuxbrew/opt/systemd/bin:/home/linuxbrew/.linuxbrew/opt/avahi/bin:/home/linuxbrew/.linuxbrew/opt/libpng/bin:/home/linuxbrew/.linuxbrew/opt/freetype/bin:/home/linuxbrew/.linuxbrew/opt/fontconfig/bin:/home/linuxbrew/.linuxbrew/opt/cairo/bin:/home/linuxbrew/.linuxbrew/opt/jpeg-turbo/bin:/home/linuxbrew/.linuxbrew/opt/libtiff/bin:/home/linuxbrew/.linuxbrew/opt/libxml2/bin:/home/linuxbrew/.linuxbrew/opt/shared-mime-info/bin:/home/linuxbrew/.linuxbrew/opt/gdk-pixbuf/bin:/home/linuxbrew/.linuxbrew/opt/fribidi/bin:/home/linuxbrew/.linuxbrew/opt/graphite2/bin:/home/linuxbrew/.linuxbrew/opt/harfbuzz/bin:/home/linuxbrew/.linuxbrew/opt/z3/bin:/home/linuxbrew/.linuxbrew/opt/llvm/bin:/home/linuxbrew/.linuxbrew/opt/spirv-llvm-translator/bin:/home/linuxbrew/.linuxbrew/opt/spirv-tools/bin:/home/linuxbrew/.linuxbrew/opt/lm-sensors/bin:/home/linuxbrew/.linuxbrew/opt/wayland/bin:/home/linuxbrew/.linuxbrew/opt/elfutils/bin:/home/linuxbrew/.linuxbrew/opt/mesa/bin:/home/linuxbrew/.linuxbrew/opt/xinput/bin:/home/linuxbrew/.linuxbrew/opt/libdatrie/bin:/home/linuxbrew/.linuxbrew/opt/pango/bin:/home/linuxbrew/.linuxbrew/opt/xauth/bin:/home/linuxbrew/.linuxbrew/opt/xkbcomp/bin:/home/linuxbrew/.linuxbrew/opt/libxcvt/bin:/home/linuxbrew/.linuxbrew/opt/nettle/bin:/home/linuxbrew/.linuxbrew/opt/xorg-server/bin:/home/linuxbrew/.linuxbrew/opt/libxkbcommon/bin:/home/linuxbrew/.linuxbrew/opt/gtk+3/bin:/home/linuxbrew/.linuxbrew/opt/gnu-which/bin:/home/linuxbrew/.linuxbrew/opt/json-glib/bin:/home/linuxbrew/.linuxbrew/opt/libnotify/bin:/home/linuxbrew/.linuxbrew/opt/numactl/bin:/home/linuxbrew/.linuxbrew/opt/alsa-lib/bin:/home/linuxbrew/.linuxbrew/opt/fftw/bin:/home/linuxbrew/.linuxbrew/opt/flac/bin:/home/linuxbrew/.linuxbrew/opt/lame/bin:/home/linuxbrew/.linuxbrew/opt/mpg123/bin:/home/linuxbrew/.linuxbrew/opt/libsndfile/bin:/home/linuxbrew/.linuxbrew/opt/m4/bin:/home/linuxbrew/.linuxbrew/opt/libtool/bin:/home/linuxbrew/.linuxbrew/opt/orc/bin:/home/linuxbrew/.linuxbrew/opt/pulseaudio/bin:/home/linuxbrew/.linuxbrew/opt/pipewire/bin:/home/linuxbrew/.linuxbrew/opt/shaderc/bin:/home/linuxbrew/.linuxbrew/opt/python@3.14/libexec/bin:/home/linuxbrew/.linuxbrew/opt/binutils/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

### Installation Command Used

```shell
brew reinstall sunshine-beta --with-cuda
```

### Additional Context

_No response_

Why do you say that?  I see errors as well relating to source locations like https://github.com/LizardByte/Sunshine/blob/master/src/platform/linux/kmsgrab.cpp#L1517 that I suspect cause issues in streaming later on. And since the warning is inside the `#ifndef` block it suggests the bottle is not built with CUDA support.

re @Jeremy-Kemp I think the build fix issue is to `brew reinstall libxorgproto` - I found the package installed but the headers not linked (`ll /home/linuxbrew/.linuxbrew/include/X11/ |& grep xorgproto` returns nothing). After reinstalling, build proceeded, but now there are many `make` errors like this:

```
cd /var/tmp/sunshine-beta-20260430-111956-fg32j2/build/third-party/libdisplaydevice/src/common && /home/linuxbrew/.linuxbrew/opt/gcc@14/bin/g++-14  -I/var/tmp/sunshine-beta-20260430-111956-fg32j2/third-party/libdisplaydevice/src/common/include -I/var/t
mp/sunshine-beta-20260430-111956-fg32j2/build/_deps/json-src/include -O3 -DNDEBUG -std=gnu++20 -MD -MT third-party/libdisplaydevice/src/common/CMakeFiles/libdisplaydevice_common.dir/logging.cpp.o -MF CMakeFiles/libdisplaydevice_common.dir/logging.cpp.o
.d -o CMakeFiles/libdisplaydevice_common.dir/logging.cpp.o -c /var/tmp/sunshine-beta-20260430-111956-fg32j2/third-party/libdisplaydevice/src/common/logging.cpp
In file included from /var/home/linuxbrew/.linuxbrew/Cellar/gcc@14/14.3.0/include/c++/14/x86_64-pc-linux-gnu/bits/gthr-default.h:35,
                 from /var/home/linuxbrew/.linuxbrew/Cellar/gcc@14/14.3.0/include/c++/14/x86_64-pc-linux-gnu/bits/gthr.h:157,
                 from /var/home/linuxbrew/.linuxbrew/Cellar/gcc@14/14.3.0/include/c++/14/bits/atomic_wait.h:40,
                 from /var/home/linuxbrew/.linuxbrew/Cellar/gcc@14/14.3.0/include/c++/14/bits/atomic_base.h:42,
                 from /var/home/linuxbrew/.linuxbrew/Cellar/gcc@14/14.3.0/include/c++/14/atomic:50,
                 from /var/tmp/sunshine-beta-20260430-111956-fg32j2/build/_deps/boost-src/libs/container/src/global_resource.cpp:123:
/var/home/linuxbrew/.linuxbrew/Cellar/gcc@14/14.3.0/include/c++/14/bits/std_mutex.h:206:32: error: cannot convert '<brace-enclosed initializer list>' to 'unsigned int' in initialization
  206 |     __gthread_cond_t _M_cond = __GTHREAD_COND_INIT;
```

Not sure if that's just a compiler flag missing/std standard flag?

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -25,6 +25,7 @@ jobs:
       platforms: ${{ steps.detect.outputs.platforms }}
       has-formulae: ${{ steps.detect.outputs.has-formulae }}
       changed-files: ${{ steps.detect.outputs.changed-files }}
+      testing-formulae: ${{ steps.detect.outputs.testing-formulae }}
       disable-failfast: ${{ steps.detect.outputs.disable-failfast }}
     steps:
       - name: Set up Homebrew
@@ -121,6 +122,7 @@ jobs:
               echo "has-formulae=false"
               echo "platforms=[]"
               echo "changed-files="
+              echo "testing-formulae="
             } >> "${GITHUB_OUTPUT}"
             exit 0
           fi
@@ -135,6 +137,10 @@ jobs:
             echo "${CHANGED_FILES}"
             echo "EOF"
           } >> "${GITHUB_OUTPUT}"
+
+          TESTING_FORMULAE=$(echo "${CHANGED_FILES}" | sed '/^$/d; s|.*/||; s|\.rb$||' | paste -sd, -)
+          echo "Formula names to test: ${TESTING_FORMULAE}"
+          echo "testing-formulae=${TESTING_FORMULAE}" >> "${GITHUB_OUTPUT}"
 
           # Check if any changed formula is in the failfast skiplist
           DISABLE_FAILFAST=false
@@ -385,7 +391,19 @@ jobs:
         if: github.event_name == 'pull_request'
         env:
           HOMEBREW_DOCKER_REGISTRY_TOKEN: ${{ steps.base64-encode.outputs.token }}
-        run: brew test-bot --fail-fast --only-formulae --root-url=https://ghcr.io/v2/lizardbyte/homebrew
+          TESTING_FORMULAE: ${{ needs.setup.outputs.testing-formulae }}
+        run: |
+          test_bot_args=(
+            --fail-fast
+            --only-formulae
+            --root-url=https://ghcr.io/v2/lizardbyte/homebrew
+          )
+
+          if [ -n "${TESTING_FORMULAE}" ]; then
+            test_bot_args+=("--testing-formulae=${TESTING_FORMULAE}")
+          fi
+
+          brew test-bot "${test_bot_args[@]}"
 
       - name: Upload bottles as artifact
         if: always() && github.event_name == 'pull_request'

--- a/Formula/c/cuda@12.9.rb
+++ b/Formula/c/cuda@12.9.rb
@@ -11,6 +11,8 @@ class CudaAT129 < Formula
   INTEL_SHA256 = "0f6d806ddd87230d2adbe8a6006a9d20144fdbda9de2d6acc677daa5d036417a".freeze
   ARM_SHA256 = "64f47ab791a76b6889702425e0755385f5fa216c5a9f061875c7deed5f08cdb6".freeze
 
+  revision 1
+
   # Set up version-specific livecheck
   CudaFormula.setup_livecheck(self)
 

--- a/Formula/c/cuda@12.9.rb
+++ b/Formula/c/cuda@12.9.rb
@@ -26,6 +26,7 @@ class CudaAT129 < Formula
 
   bottle do
     root_url "https://ghcr.io/v2/lizardbyte/homebrew"
-    sha256 cellar: :any_skip_relocation, x86_64_linux: "8dec532464bca7a342fe8a72d0aae08ce6163d15ebd32137d0037c08eff89e60"
+    sha256 cellar: :any, arm64_linux:  "9f427fcb4bcd3e5eb2736b502287474ebbfe2988bf0b9c92bfba615febad8db5"
+    sha256 cellar: :any, x86_64_linux: "10763e20564695a9dbb6573d45a4e9671e68c6cffeaa9fb0cfca0e67575fbf6d"
   end
 end

--- a/Formula/c/cuda@13.0.rb
+++ b/Formula/c/cuda@13.0.rb
@@ -26,6 +26,7 @@ class CudaAT130 < Formula
 
   bottle do
     root_url "https://ghcr.io/v2/lizardbyte/homebrew"
-    sha256 cellar: :any_skip_relocation, x86_64_linux: "99bdba97c1abdd7f83a90c6f383766c6cd6e9ac22c5da12806164929c96669cf"
+    sha256 cellar: :any, arm64_linux:  "99ed824a06e23a53939d36ea1cb2dc8ce6e84c832d7f351d4703ce49cee38317"
+    sha256 cellar: :any, x86_64_linux: "c18b139fcc9127bd657c5addf5dc876fc6dc1957e105426aa32c5c13fc96abcc"
   end
 end

--- a/Formula/c/cuda@13.0.rb
+++ b/Formula/c/cuda@13.0.rb
@@ -11,6 +11,8 @@ class CudaAT130 < Formula
   INTEL_SHA256 = "81a5d0d0870ba2022efb0a531dcc60adbdc2bbff7b3ef19d6fd6d8105406c775".freeze
   ARM_SHA256 = "93ab4c77ae2bc0f1f600ef48ccd3ff25a3203a6a6161a84511a33cbf5b5621fc".freeze
 
+  revision 1
+
   # Set up version-specific livecheck
   CudaFormula.setup_livecheck(self)
 

--- a/Formula/c/cuda@13.1.rb
+++ b/Formula/c/cuda@13.1.rb
@@ -11,6 +11,8 @@ class CudaAT131 < Formula
   INTEL_SHA256 = "24ff323723722781436804b392a48f691cb40de9808095d3e2192d0db6dfb8e4".freeze
   ARM_SHA256 = "8adcd5d4b3e1e70f7420959b97514c0c97ec729da248d54902174c4d229bfd2c".freeze
 
+  revision 1
+
   # Set up version-specific livecheck
   CudaFormula.setup_livecheck(self)
 

--- a/Formula/c/cuda@13.1.rb
+++ b/Formula/c/cuda@13.1.rb
@@ -26,6 +26,7 @@ class CudaAT131 < Formula
 
   bottle do
     root_url "https://ghcr.io/v2/lizardbyte/homebrew"
-    sha256 cellar: :any_skip_relocation, x86_64_linux: "0b78c36d6ccb5cb9f52d0d3aeb553ed7eed444595756a5d71d24bfd6a7ce1b48"
+    sha256 cellar: :any, arm64_linux:  "e4810b4ed3b724056898248f3a276d4fd90ab08d45d7232019d85fefc72b867f"
+    sha256 cellar: :any, x86_64_linux: "3d9f716aaf64a96a8b68ca53bd6bf860aad7f05b9571e678406e4a88ec29a378"
   end
 end

--- a/Formula/c/cuda@13.2.rb
+++ b/Formula/c/cuda@13.2.rb
@@ -28,5 +28,3 @@ class CudaAT132 < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "d88eb646e3d856b5e60ae68400faa762e4aeb844e56d4e30934e427d69cb9c85"
   end
 end
-
-# test rebuild

--- a/Formula/c/cuda@13.2.rb
+++ b/Formula/c/cuda@13.2.rb
@@ -26,7 +26,7 @@ class CudaAT132 < Formula
 
   bottle do
     root_url "https://ghcr.io/v2/lizardbyte/homebrew"
-    sha256 cellar: :any_skip_relocation, arm64_linux:  "e8c8578bee5c2512fec6642822222a55f658b8c68a7d580e52a492a1952bc1cd"
-    sha256 cellar: :any_skip_relocation, x86_64_linux: "d88eb646e3d856b5e60ae68400faa762e4aeb844e56d4e30934e427d69cb9c85"
+    sha256 cellar: :any, arm64_linux:  "22fb1befa9a6e7357f21df0258e790d508ee0ff86638e6205915cd65a635b33d"
+    sha256 cellar: :any, x86_64_linux: "a11ab7d98090e2bb46536db09ca874c8fc861d597d24cf002d60a8ec3e072145"
   end
 end

--- a/Formula/c/cuda@13.2.rb
+++ b/Formula/c/cuda@13.2.rb
@@ -11,6 +11,8 @@ class CudaAT132 < Formula
   INTEL_SHA256 = "5514a3fe7bcea92b25073c7c100c3e64e7961a7e1dbad6955adb8b59806053f0".freeze
   ARM_SHA256 = "38560e0c48eba793c883ea1ada6ad4c37b744cb5284034d16fd7ee57f95dda04".freeze
 
+  revision 1
+
   # Set up version-specific livecheck
   CudaFormula.setup_livecheck(self)
 

--- a/Formula/c/cuda@13.2.rb
+++ b/Formula/c/cuda@13.2.rb
@@ -28,3 +28,5 @@ class CudaAT132 < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux: "d88eb646e3d856b5e60ae68400faa762e4aeb844e56d4e30934e427d69cb9c85"
   end
 end
+
+# test rebuild

--- a/ci_config/failfast_skiplist.json
+++ b/ci_config/failfast_skiplist.json
@@ -1,7 +1,4 @@
 {
   "formulas": [
-    "cuda@12.9",
-    "cuda@13.0",
-    "cuda@13.1"
   ]
 }

--- a/lib/cuda_formula.rb
+++ b/lib/cuda_formula.rb
@@ -8,6 +8,7 @@ module CudaFormula
       depends_on "patchelf" => :build
       depends_on "cmake" => :test
       depends_on :linux
+      keg_only :versioned_formula
 
       # Default livecheck - will be overridden by setup_livecheck if CUDA_VERSION is defined
       livecheck do

--- a/lib/cuda_formula.rb
+++ b/lib/cuda_formula.rb
@@ -105,10 +105,10 @@ module CudaFormula
     library_roots = [
       libexec/"lib64",
       libexec/"extras/CUPTI/lib64",
-      *Dir[libexec/"targets/*/lib"],
+      *Dir[libexec/"targets/*/lib"].map { |path| Pathname.new(path) },
     ].select(&:directory?)
 
-    Dir[libexec/"**/*"].select { |file| File.file?(file) }.each do |file|
+    Dir[libexec/"**/*"].map { |path| Pathname.new(path) }.select(&:file?).each do |file|
       next unless File.open(file, "rb") { |f| f.read(4) == "\x7FELF".b }
 
       file_dir = file.dirname

--- a/lib/cuda_formula.rb
+++ b/lib/cuda_formula.rb
@@ -93,7 +93,8 @@ module CudaFormula
     FileUtils.rm_rf(libexec/"gds")
     FileUtils.rm_rf(libexec/"libnvvp")
 
-    FileUtils.rm_f(Dir[libexec/"bin/{ctadvisor,cuda-gdb*,cuda-uninstaller,ncu,ncu-ui,nsight*,nsys,nsys-ui,nvprof,nvvp}"])
+    FileUtils.rm_f(Dir[libexec/"bin/{ctadvisor,cuda-gdb*,cuda-uninstaller,ncu,ncu-ui,nsight*,nsys,nsys-ui}"])
+    FileUtils.rm_f(Dir[libexec/"bin/{nvprof,nvvp}"])
     FileUtils.rm_f(Dir[libexec/"pkgconfig/cuobjclient-*.pc"])
     FileUtils.rm_f(Dir[libexec/"targets/*/lib/lib{acc,cu}inj64*"])
     FileUtils.rm_f(Dir[libexec/"targets/*/lib/libcufile_rdma*"])

--- a/lib/cuda_formula.rb
+++ b/lib/cuda_formula.rb
@@ -153,7 +153,8 @@ module CudaFormula
 
   def test
     # Test that nvcc is available and can report its version
-    assert_match version.to_s, shell_output("#{bin}/nvcc --version")
+    cuda_release = version.to_s[/^\d+\.\d+/]
+    assert_match "release #{cuda_release}", shell_output("#{bin}/nvcc --version")
 
     # Test compiling a simple CUDA program
     (testpath/"test.cu").write <<~EOS

--- a/lib/cuda_formula.rb
+++ b/lib/cuda_formula.rb
@@ -72,16 +72,19 @@ module CudaFormula
 
     # Create wrapper scripts for CUDA binaries
     # This ensures they can find cuda_runtime.h, nvcc.profile, and other dependencies
+    wrapper_bin = libexec/"homebrew/bin"
+    wrapper_bin.mkpath
+
     Dir[libexec/"bin/*"].select { |f| File.file?(f) && File.executable?(f) }.each do |exe|
       binary_name = File.basename(exe)
-      (bin/binary_name).write <<~EOS
+      (wrapper_bin/binary_name).write <<~EOS
         #!/bin/bash
         export CUDA_HOME="#{libexec}"
         export PATH="#{libexec}/bin:$PATH"
         export LD_LIBRARY_PATH="#{library_paths.join(":")}:$LD_LIBRARY_PATH"
         exec "#{libexec}/bin/#{binary_name}" "$@"
       EOS
-      chmod 0755, bin/binary_name
+      chmod 0755, wrapper_bin/binary_name
     end
   end
 
@@ -130,18 +133,18 @@ module CudaFormula
         #{opt_libexec}
 
       Wrapper scripts for CUDA binaries are available at:
-        #{opt_bin}/nvcc
+        #{opt_libexec}/homebrew/bin/nvcc
 
       These wrappers automatically set CUDA_HOME for you.
 
       For CMake projects (sunshine-beta example):
         cuda_path = Formula["lizardbyte/homebrew/cuda@13.1"]
-        args << "-DCMAKE_CUDA_COMPILER=\#{cuda_path.opt_bin}/nvcc"
+        args << "-DCMAKE_CUDA_COMPILER=\#{cuda_path.opt_libexec}/homebrew/bin/nvcc"
         args << "-DCMAKE_CUDA_TOOLKIT_ROOT_DIR=\#{cuda_path.opt_libexec}"
 
       For shell/manual configuration:
         export CUDA_HOME=#{opt_libexec}
-        export PATH=#{opt_bin}:$PATH
+        export PATH=#{opt_libexec}/homebrew/bin:$PATH
         export LD_LIBRARY_PATH=#{opt_libexec}/lib64:$LD_LIBRARY_PATH
 
       This formula only installs the CUDA Toolkit (compiler and libraries).
@@ -153,9 +156,11 @@ module CudaFormula
   end
 
   def test
+    cuda_nvcc = libexec/"homebrew/bin/nvcc"
+
     # Test that nvcc is available and can report its version
     cuda_release = version.to_s[/^\d+\.\d+/]
-    assert_match "release #{cuda_release}", shell_output("#{bin}/nvcc --version")
+    assert_match "release #{cuda_release}", shell_output("#{cuda_nvcc} --version")
 
     # Test compiling a simple CUDA program
     (testpath/"test.cu").write <<~EOS
@@ -172,7 +177,7 @@ module CudaFormula
     EOS
 
     # Compile the test program
-    system bin/"nvcc", "test.cu", "-o", "test"
+    system cuda_nvcc, "test.cu", "-o", "test"
     assert_path_exists testpath/"test"
 
     # Test that CMake can find the CUDA toolkit
@@ -190,7 +195,7 @@ module CudaFormula
     # Try to configure the CMake project
     # This will fail if CMake cannot find a working CUDA compiler
     system "cmake", "-S", testpath, "-B", testpath/"build",
-           "-DCMAKE_CUDA_COMPILER=#{bin}/nvcc"
+           "-DCMAKE_CUDA_COMPILER=#{cuda_nvcc}"
 
     # Verify CMake found the CUDA toolkit
     assert_path_exists testpath/"build/CMakeCache.txt"

--- a/lib/cuda_formula.rb
+++ b/lib/cuda_formula.rb
@@ -65,6 +65,7 @@ module CudaFormula
 
     library_paths = [
       "#{libexec}/lib64",
+      *Dir[libexec/"extras/CUPTI/lib64"].map(&:to_s),
       *Dir[libexec/"targets/*/lib"].map(&:to_s),
     ].uniq
 
@@ -88,19 +89,36 @@ module CudaFormula
     # Homebrew cannot validate as portable bottle dependencies.
     FileUtils.rm_rf(Dir[libexec/"nsight-compute-*"])
     FileUtils.rm_rf(Dir[libexec/"nsight-systems-*"])
+    FileUtils.rm_rf(libexec/"extras/demo_suite")
     FileUtils.rm_rf(libexec/"gds")
+    FileUtils.rm_rf(libexec/"libnvvp")
 
-    FileUtils.rm_f(Dir[libexec/"bin/{ctadvisor,cuda-gdb*,cuda-uninstaller,ncu,ncu-ui,nsight*,nsys,nsys-ui}"])
+    FileUtils.rm_f(Dir[libexec/"bin/{ctadvisor,cuda-gdb*,cuda-uninstaller,ncu,ncu-ui,nsight*,nsys,nsys-ui,nvprof,nvvp}"])
     FileUtils.rm_f(Dir[libexec/"pkgconfig/cuobjclient-*.pc"])
+    FileUtils.rm_f(Dir[libexec/"targets/*/lib/lib{acc,cu}inj64*"])
     FileUtils.rm_f(Dir[libexec/"targets/*/lib/libcufile_rdma*"])
     FileUtils.rm_f(Dir[libexec/"targets/*/lib/libcuobjclient*"])
   end
 
   def patch_elf_rpaths
+    library_roots = [
+      libexec/"lib64",
+      libexec/"extras/CUPTI/lib64",
+      *Dir[libexec/"targets/*/lib"],
+    ].select(&:directory?)
+
     Dir[libexec/"**/*"].select { |file| File.file?(file) }.each do |file|
       next unless File.open(file, "rb") { |f| f.read(4) == "\x7FELF".b }
 
-      system "patchelf", "--set-rpath", "$ORIGIN", file
+      file_dir = file.dirname
+      rpaths = library_roots.filter_map do |root|
+        relative_path = root.relative_path_from(file_dir).to_s
+        next "$ORIGIN" if relative_path == "."
+
+        "$ORIGIN/#{relative_path}"
+      end
+
+      system "patchelf", "--set-rpath", rpaths.unshift("$ORIGIN").uniq.join(":"), file
     end
   end
 

--- a/lib/cuda_formula.rb
+++ b/lib/cuda_formula.rb
@@ -5,6 +5,7 @@
 module CudaFormula
   def self.included(base)
     base.class_eval do
+      depends_on "patchelf" => :build
       depends_on "cmake" => :test
       depends_on :linux
 
@@ -55,9 +56,17 @@ module CudaFormula
            "--toolkit",
            "--toolkitpath=#{libexec}"
 
+    prune_nonportable_payload
+    patch_elf_rpaths
+
     # NOTE: We do not symlink lib64 or include to avoid conflicts and hacks
     # Dependent formulae should use CMAKE_CUDA_TOOLKIT_ROOT_DIR=#{libexec}
     # or set CUDA_HOME=#{libexec} to find libraries and headers
+
+    library_paths = [
+      "#{libexec}/lib64",
+      *Dir[libexec/"targets/*/lib"].map(&:to_s),
+    ].uniq
 
     # Create wrapper scripts for CUDA binaries
     # This ensures they can find cuda_runtime.h, nvcc.profile, and other dependencies
@@ -67,10 +76,31 @@ module CudaFormula
         #!/bin/bash
         export CUDA_HOME="#{libexec}"
         export PATH="#{libexec}/bin:$PATH"
-        export LD_LIBRARY_PATH="#{libexec}/lib64:$LD_LIBRARY_PATH"
+        export LD_LIBRARY_PATH="#{library_paths.join(":")}:$LD_LIBRARY_PATH"
         exec "#{libexec}/bin/#{binary_name}" "$@"
       EOS
       chmod 0755, bin/binary_name
+    end
+  end
+
+  def prune_nonportable_payload
+    # These bundled GUI/profiler/debugger/GDS tools link host libraries that
+    # Homebrew cannot validate as portable bottle dependencies.
+    FileUtils.rm_rf(Dir[libexec/"nsight-compute-*"])
+    FileUtils.rm_rf(Dir[libexec/"nsight-systems-*"])
+    FileUtils.rm_rf(libexec/"gds")
+
+    FileUtils.rm_f(Dir[libexec/"bin/{ctadvisor,cuda-gdb*,cuda-uninstaller,ncu,ncu-ui,nsight*,nsys,nsys-ui}"])
+    FileUtils.rm_f(Dir[libexec/"pkgconfig/cuobjclient-*.pc"])
+    FileUtils.rm_f(Dir[libexec/"targets/*/lib/libcufile_rdma*"])
+    FileUtils.rm_f(Dir[libexec/"targets/*/lib/libcuobjclient*"])
+  end
+
+  def patch_elf_rpaths
+    Dir[libexec/"**/*"].select { |file| File.file?(file) }.each do |file|
+      next unless File.open(file, "rb") { |f| f.read(4) == "\x7FELF".b }
+
+      system "patchelf", "--set-rpath", "$ORIGIN", file
     end
   end
 

